lib/metrics.js

```diff
@@ -24,7 +24,7 @@ class Metrics {
       // don't let metrics keep the process alive
       this.id.unref()
     }
-    return this.state = 'started'
+    return this.state = 'started' // eslint-disable-line no-return-assign
```

Comment:
Minor and maybe stylistic: since it's deemed not as readable, can this be turned into two statements, an actual assign followed by a return?

Reply:
Yep. But I'd rather not touch *any* of the reassigns in this pass.